lib/adminlte-rails/source_file.rb

```diff
@@ -93,54 +67,24 @@ def fetch_images(remote, tag)
 
   def fetch_stylesheets(remote, tag)
     # Get Css files
-    css_path = "#{remote}/raw/#{tag}/css"
-    get "#{css_path}/bootstrap.css", 'stylesheets/bootstrap.scss'
-    get "#{css_path}/AdminLTE.css", 'stylesheets/admin-lte.css'
-    get "#{css_path}/ionicons.css", 'stylesheets/ionicons.scss'
-    get "#{css_path}/font-awesome.css", 'stylesheets/font-awesome.scss'
+    css_path = "#{remote}/raw/#{tag}/dist/css"
 
+    get "#{css_path}/AdminLTE.css", 'stylesheets/admin-lte.css'
+    get "#{css_path}/skins/skin-purple.css", 'stylesheets/skin-purple.css'
+    get "#{css_path}/skins/skin-blue.css", 'stylesheets/skin-blue.css'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.